roles/controller_casc_from_aap/templates/current_projects_improved.j2

```diff
@@ -0,0 +1,20 @@
+---
+controller_projects:
+{% for project in current_pj_asset_value %}
+  - name: "{{ project.name }}"
+    description: "{{ project.description }}"
+    organization: "{{ project.organization.name | default('ToDo: The project \'' + project.name + '\' must have an organization') }}"
```

Comment:
organization: "{{ project.summary_fields.organization.name | default('ToDo: The project \'' + project.name + '\' must have an organization') }}"